includes/templates/reader-activation-emails/verification.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Homepage Pattern.
+ *
+ * @package Newspack
+ */
+
+// phpcs:disable WordPressVIPMinimum.Security.Mustache.OutputNotation
+
+return array(
+	'post_title'   => __( 'Your verification link', 'newspack' ),
+	'post_content' => '<!-- wp:site-logo {"align":"center"} /-->
+
+  <!-- wp:heading -->
+  <h2>Hi there!</h2>
+  <!-- /wp:heading -->
+
+  <!-- wp:paragraph -->
+  <p>To manage your account, please verify your email address by clicking the link below:</p>
```

Comment:
Shouldn't all boilerplate copy here be run through `__()`?

Reply:
Done in 635bbfe599297087e2c17bcc1aaf3ecad1c84f02